requirements-sdp.txt

```diff
@@ -1 +1 @@
-# Nothing here right now
\ No newline at end of file
+asdf==2.5.2
```

Comment:
We don't want anything here right now.  Eventually this file will get populated via `pip freeze` for build 7.5.